.circleci/config.yml

```diff
@@ -63,6 +63,19 @@ common-steps:
         echo $PKG_NAME > ~/packaging/sd_package_name
         echo 'export PKG_NAME=$(cat ~/packaging/sd_package_name)' >> $BASH_ENV
 
+  - &clonesecuredropexport
+    run:
+      name: Clone the repository to be packaged
+      command: |
+        mkdir ~/packaging && cd ~/packaging
+        git clone https://github.com/freedomofpress/securedrop-export.git
+        # Checkout WIP branch
+        cd securedrop-export && git checkout add-makefile-ci
```

Comment:
`add-makefile-ci` was merged into master (CI is failing due to a subsequent upstream module deletion), so this is probably unnecessary

Reply:
good catch, fixed (I made a mistake while rebasing earlier)